docs/cis_self_assessment.md

```diff
@@ -2214,58 +2012,39 @@ Kubelets can automatically manage the required changes to iptables based on how
 
 **Result:** Pass
 
-**Remediation:**
-If using a Kubelet config file, edit the file to set `makeIPTablesUtilChains`: `true`.
-If using command line arguments, edit the kubelet service file
-`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` on each worker node and
-remove the `--make-iptables-util-chains` argument from the
-`KUBELET_SYSTEM_PODS_ARGS` variable.
-Based on your system, restart the kubelet service. For example:
+**Audit**
+Run the below command on the master node.
 
 ```bash
-systemctl daemon-reload
-systemctl restart kubelet.service
-```
-
-**Audit:**
-
-```
-/bin/ps -fC kubelet
-```
-
-**Audit Config:**
-
-```
-/bin/cat /var/lib/kubelet/config.yaml
+/bin/ps -ef | grep kubelet | grep -v grep
 ```
 
-**Expected result**:
+Verify there are no results returned.
 
-```
-'true' is equal to 'true' OR '--make-iptables-util-chains' is not present
-```
+**Remediation:**
+By default, RKE2 does not set the `--make-iptables-util-chains` argument. No manual remediation needed.
 
 
 #### 4.2.8
-Ensure that the --hostname-override argument is not set (Not Scored)
+Ensure that the `--hostname-override` argument is not set (Not Scored)
 <details>
 <summary>Rationale</summary>
 Overriding hostnames could potentially break TLS setup between the kubelet and the apiserver. Additionally, with overridden hostnames, it becomes increasingly difficult to associate logs with a particular node and process them for security analytics. Hence, you should setup your kubelet nodes with resolvable FQDNs and avoid overriding the hostnames with IPs.
 </details>
 
-**Result:** Pass
+**Result:** Not Applicable
 
 **Remediation:**
 
 
 #### 4.2.9
-Ensure that the --event-qps argument is set to 0 or a level which ensures appropriate event capture (Not Scored)
+Ensure that the `--event-qps` argument is set to 0 or a level which ensures appropriate event capture (Not Scored)
 <details>
 <summary>Rationale</summary>
 It is important to capture all events and not restrict event creation. Events are an important source of security information and analytics that ensure that your environment is consistently monitored using the event data.
 </details>
 
-**Result:** Pass
+**Result:** Not Applicable
 
 **Remediation:**
```

Comment:
Say something like "See CIS Benchmark guide for further details on configuring this."